src/sentry/api/serializers/models/project.py

```diff
@@ -176,8 +171,43 @@ def measure_span(op_tag):
                 )
                 if stats:
                     serialized["stats"] = stats[project.id]
+                if transaction_stats:
+                    serialized["transactionStats"] = transaction_stats[project.id]
         return result
 
+    def get_stats(self, project_ids, query):
+        # we need to compute stats at 1d (1h resolution), and 14d
+        segments, interval = STATS_PERIOD_CHOICES[self.stats_period]
+        now = timezone.now()
+
+        params = {
+            "project_id": project_ids,
+            "start": now - ((segments - 1) * interval),
+            "end": now,
+        }
+        if self.environment_id:
+            query = u"{} environment:{}".format(query, self.environment_id)
+        stats = discover.top_events_timeseries(
```

Comment:
Could we use `discover.query` to create snuba results first (or fake it since we have the projects already), and share that for both transaction + non_transaction with the `top_events` parameter?